cmd/cometbft/commands/inspect.go

```diff
@@ -58,7 +58,7 @@ func runInspect(cmd *cobra.Command, _ []string) error {
 	if err != nil {
 		return err
 	}
-	blockStore := store.NewBlockStore(blockStoreDB)
+	blockStore := store.NewBlockStore(blockStoreDB, store.BlockStoreOptions{Metrics: store.NopMetrics()})
```

Comment:
```suggestion
	blockStore := store.NewBlockStore(blockStoreDB, store.BlockStoreOptions{})
```

We can remove `NopMetrics` because it's a default